templates/home/_fsf_rust.html

```diff
@@ -4,14 +4,44 @@ <h4>Why are snaps good for Rust projects?</h4>
     <ul>
       <li>Bundle all the runtime requirements, with support for multiple architectures.</li>
       <li>Simplify installation instructions, regardless of distribution, to snap install myrustapp.</li>
-      <li>Directly control the delivery of automatic application updates.</li>
       <li>Extremely simple creation of services.</li>
       <li>No complex installation shell scripts to maintain.</li>
     </ul>
+	<p>Programming in Rust can sometimes be difficult, especially when one of the crates in your project changes. Snaps let you bundle all the necessary dependencies without having to worry about breakages.</p>
```

Comment:
>"Programming in Rust can sometimes be difficult, especially when one of the crates in your project changes."

Why do changes to crates make programming in Rust difficult?

>"Snaps let you bundle all the necessary dependencies without having to worry about breakages."

Do Rust applications typically use C library dependencies?

Reply:
From what I'm able to discern, updating a dependency in cargo.toml from one version to another can lead to the crate being broken (in some cases), and this cannot necessarily be easily fixed unless you maintain all the crates.

Reply:
How do snaps improve upon that?

Reply:
I have pushed a revised template file. At the moment, I do not have any additional information to add on the Rust-specific advantages to the landing page of the Rust FSF guide. I will study this topic some more.

The snap builds and installs correctly (based on one of Alan's Top 100 projects).